### PR TITLE
removed old_region from company #139602987

### PR DESCRIPTION
**Pull request by patmbolger**
PT Story: https://www.pivotaltracker.com/story/show/139602987

Changes proposed in this pull request:
1. Removed `old_region` attribute from Company model
2. Adjusted prod and test code
3. Adjusted rake task

Ready for review:
@weedySeaDragon 
@thesuss

### CODE REVIEW

File: app/models/company.rb [hunk 1]
```diff
@@ -49,8 +48,7 @@ def error_if_has_accepted_applications?
 
 
   def complete?
```
Comment (weedySeaDragon) [inline app/models/company.rb]: This method is no longer used anywhere.  Should we delete it?

File: app/helpers/companies_helper.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module CompaniesHelper
 
   def company_complete? company
-    company&.name.blank? ||
-    (company&.region.blank? && company&.old_region.blank?) ? false : true
+    return false if ! company
```
Comment (weedySeaDragon) [inline app/helpers/companies_helper.rb]: I'm not sure this is used anywhere.  When I did a search in the code for `company_complete?`  I only found it being used in the specs and features.

We now only return `complete` companies from the database so there's no need to filter them in the views (via a helper).  So perhaps we can remove this method altogether?

If we do keep it, what about changing this to:

```
company.nil? ? false : company.complete?
```

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
2 minor questions and a suggestion.

File: app/helpers/companies_helper.rb (hunk 2 above)
Comment (patmbolger) [reply app/helpers/companies_helper.rb]: You're right about both methods.

The only reason I might think to retain these is that I imagine in the future we'll be asked to create a view showing _all_ companies, regardless of "complete" status.  For instance, to allow for deleting abandoned efforts to create a company (which are cluttering up the DB with useless records).

However, that is not an argument to retain these methods.  We can cross that bridge when we come to it.  (should have given a cliché trigger warning ...)

I will remove the two methods.

**Review by weedySeaDragon — APPROVED** [review]

**Pull request merged**